### PR TITLE
Added tlbflush test in avocado

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: Praveen K Pandey praveen@linux.vnet.ibm.com
Added tlbflush test

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
Please find job log in below link 

https://paste.fedoraproject.org/408239/19166914/

File: kernel/tlbflush.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            out = self.run()
+            self.perf_json.append({'Test time' + str(ite): out})
+
+        output_path = os.path.join(self.outputdir, "perf.json")
+        json.dump(self.perf_json, open(output_path, "w"))
+
+    def run(self):
+        out = process.system_output('%s ' % os.path.join(
+            self.srcdir, 'tlbflush'), '-n ' + str(self.nr_entries) +
+            ' -t' + str(self.nr_threads))
```
Comment (apahim) [inline kernel/tlbflush.py:89]: This line is a little confuse, with `%s` and `+`, `join()`... could you make it easier to read? Something like:

``` python
def run(self):
    tlbflush = os.path.join(self.srcdir, 'tlbflush')
    cmd = '%s -n %s -t %s' % (tlbflush, self.nr_entries, self.nr_threads)
    return process.system_output(cmd)
```

Other than that, looks good.

Comment (PraveenPenguin) [inline kernel/tlbflush.py:89]: @apahim  sure will do , will raise new PR 

Thanks

**Comment by PraveenPenguin** [comment]
@apahim raise a new PR #142 with changes ,  Please have a look 

Thanks

**Pull request closed without merging**